CS342MemberViews/Sources/AlexisLowber/AlexisLowber.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  SwiftUIView.swift
+//  
+//
+//  Created by Alexis Lowber on 1/24/23.
+//
```

Comment:
The REUSE specification requires all files to include licensing information. Please adjust your headers to follow the following format:
```suggestion
//
// This source file is part of the CS342 2023 Application project
//
// SPDX-FileCopyrightText: 2023 Stanford University
//
// SPDX-License-Identifier: MIT
//
```

Reply:
Thanks for the review. I'll update my branch and add a creative component!